jobs/async-upload/samples/sample_job_s3_to_oci.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: my-async-upload-job
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: async-upload
+          image: kubeflow/model-registry-async-job:local
+          env:
+            # ---- Source ----------
+            - name: MODEL_SYNC_SOURCE_TYPE
+              value: "s3"
+            - name: MODEL_SYNC_SOURCE_AWS_BUCKET
+              value: "default"
```

Comment:
seen new addition of Manifest; just to be sure, the actual Manifest need to contain mounting of the Secrets, which contains keys as specified in ADR; correct?

it doesn't hurt to have a first sample, but we need a sample which complies with what we defined (ie in parlance "mounting the Connections(Secrets)")